package.json

```diff
@@ -9,7 +9,7 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.35",
     "@fortawesome/free-solid-svg-icons": "^5.13.0",
     "@fortawesome/react-fontawesome": "^0.1.9",
-    "@metacell/geppetto-meta-client": "1.0.0-final",
+    "@metacell/geppetto-meta-client": "file:.yalc/@metacell/geppetto-meta-client",
```

Comment:
it would be ideal to have a dev-package.json so that we swap them during the installation if we want to use geppetto-meta from sources.